secure_message/api_mocks/party_service_mock.py

```diff
@@ -9,16 +9,23 @@ class PartyServiceMock:
 
     def get_business_details(self, ru):
         """Return mock business details"""
-        business_details = self._business_details.get(ru)
-        if business_details is None:
+        business_details = []
+        for ru_id in self._business_details:
+            if ru_id in ru:
+                business_details.append(self._business_details.get(ru_id))
```

Comment:
This could be a list comp:
```
new_list = []
for i in old_list:
    if filter(i):
        new_list.append(expressions(i))
```

to:
`new_list = [expression(i) for i in old_list if filter(i)]`

Reply:
Done